core/modules/layout/includes/block.hero.inc

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ *  BlockHero extends BlockText
+ *
+ *  This class allows us to create hero blocks with image backgrounds.
+ */
+class BlockHero extends BlockText {
+  /**
+   * {@inheritdoc}
+   */
+  function __construct($plugin_name, array $data) {
+    parent::__construct($plugin_name, $data);
+
+    $this->settings += array(
+      'image' => '',
+      'image_path' => '',
+    );
+  }
+
+  /**
+   *  Sets title text on draggable block panel in Layout builder.
+   */
+  function getAdminTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Hero block');
+  }
+
+  /**
+   *  Builds the block's configuration form.
+   */
+  function form(&$form, &$form_state) {
+    parent::form($form, $form_state);
+
+    $upload_validators = array(
+      'file_validate_extensions' => array('jpg jpeg png gif'),
+      'file_validate_image_resolution' => array('3200x1600', '1200x600'),
+    );
+    $upload_description = theme('file_upload_help', array(
+      'description' => t('The uploaded image will be displayed as a background.'),
```

Comment:
I think we're getting into the realm of way too much 'help' text. I don't think the description I added actually adds any value since the field label already says it's a background image. 
![screen shot 2016-09-09 at 12 17 30 am](https://cloud.githubusercontent.com/assets/397895/18378629/fb453e96-7622-11e6-9e1b-bedab597b8ed.png)